test/jobs/add_guest_to_send_grid_job_test.rb

```diff
@@ -1,12 +1,13 @@
 require 'test_helper'
 
-class AddGuestToSendGridJobTest < ActiveJob::TestCase
+class AddGuestToSendGridJobTest < ActiveSupport::TestCase
   test 'it adds the guest to send grid' do
+    job = AddGuestToSendGridJob.new
     guest = SendGridClient::Guest.user(valid_email)
 
     SendGridClient.any_instance.expects(:add_user).with(guest)
 
-    AddGuestToSendGridJob.perform_now(valid_email)
+    job.perform(valid_email)
```

Comment:
Since this job is derived from the sidekiq::Worker class we use the `perform` and `perform_async` methods?

Does `perform_now` and `perform_later` hook into the new sidekiq::Worker?

Reply:
> Since this job is derived from the sidekiq::Worker class we use the perform and perform_async methods?

Correct!

> Does perform_now and perform_later hook into the new sidekiq::Worker?

No. Because these classes no longer descend from ActiveJob, they do not have now/later methods. That's the ActiveJob API. Sidekiq workers have `perform` (now) and `perform_async` (later).